tests/base.py

```diff
@@ -55,6 +62,12 @@ def setUpClass(cls):
                   'database': config['Database']['name'],
                   'host': config['Database']['host'],
                   'port': config['Database']['port']}
+        if not TestDatabaseCaseBase.db_created:
+            try:
+                Database.create(**cls.db_kwargs)
+            except DatabaseExists as e:
+                pass
+            TestDatabaseCaseBase.db_created = True
```

Comment:
Why is this needed? The method `setUpClass` is only called once. Moreover, tests suite must keep the system as it was before running the tests. I think the database should be deleted once the tests finish (dropping the database inside `tearDownClass` method.